.github/workflows/scripts/translation_tag_check.py

```
@@ -0,0 +1,305 @@
+        # Scan both src and plugins directories by default
+        found_source = False
+        for directory in ["src", "plugins"]:
+            dir_path = Path(directory)
+            if dir_path.exists() and dir_path.is_dir():
+                targets.extend(dir_path.rglob("*"))
+                found_source = True
+
+        if not found_source:
+            raise FileNotFoundError(
+                "Default 'src' or 'plugins' directory not found"
+            )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)